Week2/homework/Exercise5.js

```diff
@@ -0,0 +1,23 @@
+const myHobbies = [
+    'Meditation',
+    'Reading',
+    'Programming',
+    'Hanging out with friends',
+    'Going to the gym',
+  ];
+ 
+  var list = document.createElement('ul');
+  
+
+    myHobbies.map(funct);
```

Comment:
remember that map returns an array of items. If you don't intend to do anything with this result then maybe you should have used the .forEach method